deploy-tool/cert/kubernetes_storage_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func TestCleanKey(t *testing.T) {
-	prefix := "cm.k8s."
+	prefix := "acme."
```

Comment:
I assume this test was failing without this change? Always good to prove the test works!

Reply:
Yep!